### PR TITLE
Add German PDF and Fix print German

### PR DESCRIPTION
**Pull request by Hexaflexagon**

### CODE REVIEW

**Comment by Hexaflexagon** [comment]
I added a geerman PDF file and changed the cable translation

**Comment by brugnara** [comment]
@reneroth & @FroznMine could you please check this PR as I don't speak German? 👀 Many thanks 👍  

Also, thanks in advice for your effort, @Hexaflexagon

**Comment by FroznMine** [comment]
Looks good to me. Thanks for finding these errors!

File: german/index.html [hunk 1]
```diff
@@ -360,11 +360,13 @@ <h2 style="margin: 1.2em auto; text-decoration: none; line-height: 1em; font-siz
 			<ul>
 			<li>Auf diesem Modul gibt es mehrere Paneele auf denen Kabeln sind, aber nur ein Paneel ist zur Zeit sichtbar. Wechseln Sie zum nächsten Paneel, indem Sie den Pfeil-nach-unten-Knopf drücken, und zum vorigen mit dem Pfeil-nach-oben-Knopf.</li>
 			<li>Wechseln Sie nicht zum nächsten Paneel, wenn Sie nicht sicher sind, dass Sie alle notwendigen Kabel auf dem aktuellen Paneel durchgeschnitten haben.</li>
-			<li>Wählen Sie die durchzuschneidenden Kabel anhand der folgenden Tabelle aus. Kabelvorkommen sind kumulativ über die gesamten Paneele des Moduls.</li>
+			<li>Wählen Sie die durchzuschneidenden Kabel anhand der folgenden Tabelle aus. Ist das auf dem Modul zusehende Kabel mit dem in der Tabelle angegebenen Buchstaben verbunden, so schneiden Sie das Kabel durch. Kabelvorkommen sind kumulativ über die gesamten Paneele des Moduls.</li>
```
Comment (reneroth) [inline german/index.html:363]: zusehende - zu sehende

**Comment by reneroth** [comment]
caught a tiny typo :) thanks for adding a PDF!

**Comment by MergingMonkey** [comment]
At least we have a PDF (and thus a usable printable version), but the margins are pretty big (or the scaling is too low). Maybe you can reduce them?

**Comment by brugnara** [comment]
the typo @reneroth found is fixed? Just to not do a commit for a missing space :)

**Comment by brugnara** [comment]
ping

**Comment by dakira** [comment]
@brugnara the typo is not fixed, yet. @Hexaflexagon

**Review by dakira — CHANGES_REQUESTED** [review]
looks good to me, except for the typo @reneroth already found

**Pull request closed without merging**